docs/semgrep-app/getting-started-with-semgrep-app.md

```diff
@@ -214,28 +214,33 @@ Scanning a repository from SCM providers such as GitHub or GitLab ensures that S
 1. Ensure you are signed in to Semgrep App.
 2. Click **Projects** on the left sidebar.
 3. Optional: If you do not see the repository you want to add in the **Projects** page of Semgrep app, follow the steps in the succeeding sections to ensure that Semgrep App can detect the repository.
-4. Click **Scan new project**, and then select which repository provider Semgrep to integrate with.
-5. For **GitHub Actions**:
-    1. Click **Add CI Job **next to the name of the project to add.
-    2. Optional: If you do not see the repository you want to add, follow the succeeding guide on **detecting GitHub repositories**.
-    3. Click **Commit file**. This commits a `semgrep.yml` file containing pertinent scan parameters, such as schedule, what branch to scan on, and so on.
-    4. Semgrep App generates a token composed of a **Secret name** and **Secret value**. Copy and paste these into your repository's settings. Links are provided by Semgrep App.
-    5. Click **The secret's there, continue**.
-    6. Semgrep App creates a file for the CI job. Select toggles for desired features, such as scanning on a schedule.
-6. For **GitLab CI/CD**:
-    1. Create a Semgrep App token by clicking **Settings > Tokens > Create new token**, then return to the project setup by clicking on the back arrow on your browser.
+4. Click **Scan new project**, and then click **Run scan in CI**.
+5. Select which repository provider Semgrep to integrate with.
+6. For **GitHub Actions**:
+    1. Select the checkbox next to the name of the projects to add.
+    2. Optional: If you do not see the repository you want to add, adjust [GitHub Application's Repository Access](https://github.com/settings/installations) configuration. See [Detecting GitHub repositories](#detecting-github-repositories) for more information.
+    3. Click **Bulk add CI jobs**.
+    4. Click **Commit file**. This commits a `semgrep.yml` file containing pertinent scan parameters, such as schedule, what branch to scan on, and so on.
+    5. Semgrep App generates a token composed of a **Secret name** and **Secret value**. Copy and paste these into your repository's settings. Links are provided by Semgrep App.

```

Comment:
This step number 5. does not seem to be true anymore - can we delete this @milanwilliams?
```suggestion
```

Reply:
@adamkvitek Just curious, how does it behave right now in your personal org?

Reply:
@s-santillan I just see [these steps ](https://github.com/returntocorp/semgrep-docs/pull/688/files#diff-6f8bfa5fc733c7745e1fd59c0ac3d97925023b89a4baa1b32480b4135867dd3eR219-R224) now - I will try in one more testing organization I have (maybe my `adamkvitek` org already got some additional permissions).

Reply:
Permissions for what? For Github actions, the only permissions you need are here:
[https://semgrep.dev/docs/semgrep-ci/running-semgrep-ci-with-semgrep-app/#other-ci-providers-environment-variables-setup](https://semgrep.dev/docs/semgrep-ci/running-semgrep-ci-with-semgrep-app/#other-ci-providers-environment-variables-setup)